### PR TITLE
Pull Request for Aquaman

### PR DESCRIPTION
**Pull request by MariovdBerg**
Aquaman a.k.a. Jakob Christiaanse

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (jakobchristiaanse) [inline inflammation/models.py:36]:
```suggestion
def daily_std(data):
```

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```
Comment (jakobchristiaanse) [inline inflammation/models.py:44]: There is a numpy function np.std() which you can use, much simpler than computing the std yourself

File: inflammation-analysis.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (jakobchristiaanse) [inline inflammation-analysis.py:24]: deze begriep ik nie

**Review by jakobchristiaanse — COMMENTED** [review]
reasonable work but not great Mario

File: inflammation-analysis.py (hunk 3 above)
Comment (MariovdBerg) [reply inflammation-analysis.py:24]: I will change the function return to correspond with the used format.

File: inflammation/models.py (hunk 2 above)
Comment (MariovdBerg) [reply inflammation/models.py]: I will certainly change the function to use this!

**Pull request merged**